R/blend_rgba_float.R

```diff
@@ -0,0 +1,35 @@
+#' blend_rgba_float
+#'
+#' Blend two RGBA matrices.
+#'
+#' @param fRGBA_1 Float RGBA matrix (`red`, `green`, `blue` and `alpha` channels, dimension nxmx4, values ~ 0-1).
+#'
+#' @param fRGBA_2 The same as fRGBA_1 parameter.
+#'
+#' @return Float RGBA matrix.
+#'
+#' @export
+#' @useDynLib scattermore, .registration=TRUE
+blend_rgba_float <- function(fRGBA_1, fRGBA_2)
+{
+    dim_RGBA <- 4
```

Comment:
tbh this might make a good global constant

Reply:
hmm, i see that global variables/constants in R are evil (discussions)... should we do it nevertheless?

Reply:
actually functions are global variables there too. :shrug: 

to make it a bit less variablish you can just do something like
```r
_RGBA_channels <- function() 4
```